test/unit/framework/mesh/mesh_vector_test.cc

```diff
@@ -0,0 +1,232 @@
+#include "test/unit/opensn_unit_test.h"
+#include "framework/mesh/mesh_vector.h"
+
+using namespace opensn;
+
+class MeshVectorTest : public OpenSnUnitTest
```

Comment:
I would rename this to `Vector3Test`. The class being tested is `Vector3` and the file will be renamed soon(tm).